File: stacks/dotnet_8/scaffold/src/App/Greeter.cs
namespace App;

/// <summary>
/// Provides greeting functionality.
/// </summary>
public static class Greeter
{
    /// <summary>
    /// Returns a greeting message for the given name.
    /// </summary>
    /// <param name="name">The name to greet.</param>
    /// <returns>A greeting message.</returns>
    public static string Greet(string name)
    {
        return $"Hello, {name}!";
    }
}

File: stacks/java_21_gradle/scaffold/src/main/java/app/Greeter.java
package app;

/**
 * Provides greeting functionality.
 */
public class Greeter {
    
    /**
     * Returns a greeting message for the given name.
     *
     * @param name the name to greet
     * @return a greeting message
     */
    public static String greet(String name) {
        return "Hello, " + name + "!";
    }

    public static void main(String[] args) {
        System.out.println(greet("World"));
    }
}
